llama_index/vector_stores/postgres.py

```diff
@@ -117,6 +118,7 @@ class PGVectorStore(BasePydanticVectorStore):
     _session: Any = PrivateAttr()
     _async_engine: Any = PrivateAttr()
     _async_session: Any = PrivateAttr()
+    _perform_setup: bool = PrivateAttr(default=True)
```

Comment:
Is there a reason to have both a private attribute and a class attribute? We can probably pick one or the other?  I had a similar PR [here](https://github.com/run-llama/llama_index/pull/8633) btw

Reply:
Whoops.  Made this a class attribute now.